migrations/1676395230930_inscriptions.ts

```diff
@@ -6,15 +6,27 @@ export const shorthands: ColumnDefinitions | undefined = undefined;
 export function up(pgm: MigrationBuilder): void {
   pgm.createTable('inscriptions', {
     id: {
-      type: 'serial',
+      type: 'bigserial',
```

Comment:
We should add an eslint rule to warn/error on any detected usage of the 32bit-int types in postgres. It's unintuitive and has caused a problems so many times in a lot of our projects

Reply:
Great point, would be super useful on the API toolkit I think, I'll open an issue there